docs/operator-guide/customize_deployment.md

```diff
@@ -0,0 +1,52 @@
+# Customize Deployment
+
+When deploying applications into envirnments, it might be necessary to automate some post-deployment procedures, such as testing, configuring, removing old information from the environment, etc. For these purposes, the custom deployment feature was implemented.
```

Comment:
we also support pre-deployment customization, let's just say about customization of deployment in general

Reply:
Done